src/components/SelectListForm.jsx

```diff
@@ -3,17 +3,19 @@ import { useState } from 'react';
 
 export default function SelectListForm({
 	data,
-	listPath,
 	areListsLoading,
+	listName,
 	setListPath,
 }) {
-	const [selectedList, setSelectedList] = useState('');
+	const [selectedList, setSelectedList] = useState(listName);
```

Comment:
You can set a default state here for if no list is selected, something like "No list Selected"